enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -0,0 +1,353 @@
+---
+title: mixed-cpu-node-plugin
+authors:
+  - "@Tal-or"
+reviewers: 
+  - "@MarSik"
+  - "@yanirq"
+  - "@ffromani"
+  - "@jmencak"
+  - "@mrunalp"
+  - "@dhellmann"
+approvers: 
+  - "@mrunalp"
+  - "@dhellmann"
```

Comment:
```suggestion
```

I'm not going to have time to act as approver for this one. If @mrunalp does not have time, perhaps @rphillips will.

Reply:
I've added @rphillips , thank you!